### DISCUSSION TITLE
Fair Launching a PCO Asset

### DISCUSSION THREAD

**Discussion by gravenp** [RadicalxChange/salsa-nft#7] [Unsolved Questions]
With a greenfield PCO implementation, what are the "best" ways to initially allocate the scarce resource/asset subject to PCO? What factors would make the allocation strategy different for a PCO asset vs "normal" NFT drop?

- First come, first serve - probably only makes sense if supply dramatically outpaces supply
- [Reverse Dutch auction ](https://en.wikipedia.org/wiki/Dutch_auction)
- [Vickery auction](https://en.wikipedia.org/wiki/Vickrey_auction)
- Raffles
- Whitelists
- [Candle auction](https://en.wikipedia.org/wiki/Candle_auction)

**Comment by laplace-daemon** [comment]
If the asset is always for sale why is there a need to fair launch it? It's always fair as there's no advantage in acquiring the asset first.

**Reply by gravenp** [reply]
If a PCO asset is just being created, there is no previous licensor and no always for sale price when the market opens up (unless one of these mechanisms is used). 

So there's a race incentive to be the first to claim the asset for free and set a for sale price greater than 0 (assuming market demand). If someone genuinely values the asset more but is slower to claim, the first claimant will get the spread between their 0 cost claim and the first sale price with minimal fee payments (very little time has passed).
